src/pages/journal-vouchers/forms/JournalVoucherForm.js

```diff
@@ -112,6 +123,13 @@ export default function JournalVoucherForm({ labels, maxAccess, recordId }) {
       disabled: !editMode
     }
   ]
+  useEffect(() => {
+    data?.errorMessage && stackError({ message: data?.errorMessage })
+  }, [data?.errorMessage])
+
+  useEffect(() => {
```

Comment:
Remove

Reply:
i need this

Reply:
Why? Is there another way to have outside formik (not state)